sqlite-db/src/lib.rs

```diff
@@ -277,39 +269,20 @@ impl Connection {
         let mut conn = self.inner.acquire().await?;
         let mut db_tx = conn.begin().await?;
 
-        let cache_key = (TypeId::of::<C>(), id);
-        let aggregate = std::any::type_name::<C>();
-
-        let cfd = match self.aggregate_cache.remove(&cache_key).await {
```

Comment:
Even if the cache remains, I think that using a concurrent hashmap should be unnecessary. Something DashMap should probably suffice, as the CFD guard is never very long, so the .awaits probably add unnecessary overhead.

This might sound surprising, given that acquiring a lock is generally considered blocking. However, when the critical section is so short as it is here, an async lock does more harm than good. See the docs of [`tokio::sync::Mutex`](https://docs.rs/tokio/latest/tokio/sync/struct.Mutex.html) for more on this concept.